.github/workflows/roll_torch_mlir.yml

```diff
@@ -0,0 +1,60 @@
+name: Roll PyTorch
+
+on:
+  schedule:
+    - cron: '0 13 * * *'
+  workflow_dispatch:
+
+jobs:
+  build_linux:
+    name: roll torch-mlir
+    needs: setup
+    runs-on:
+      - self-hosted
+      - runner-group=${{ needs.setup.outputs.runner-group }}
+      - environment=${{ needs.setup.outputs.runner-env }}
+      - cpu
+      - os-family=Linux
+
+    # Don't run this in everyone's forks.
+    if: github.repository == 'open-xla/iree'
+
+    steps:
+
+      - name: Prepare workspace
+        run: |
+          # Clear the workspace directory so that we don't run into errors about
+          # existing lock files.
+          sudo rm -rf $GITHUB_WORKSPACE/*
+
+      - name: Get iree
+        uses: actions/checkout@v3
+        with:
+          submodules: 'true'
+          token: ${{ secrets.WRITE_ACCESS_TOKEN }}
+          ref: main
+          fetch-depth: 0
+
+      - name: update torch_mlir submodule
+        run: |
+          git config --local user.email "iree-github-actions-bot@google.com"
+          git config --local user.name "Roll torch-mlir"
+          git switch -c "rolltorchmlir_${{ github.run_number }}"
+          git submodule update --init --recursive third_party/torch-mlir
+          cd third_party/torch-mlir
+          git fetch --tags origin
+          git checkout origin/main
+          cd ../..
+          git add third_party/torch-mlir
+          git commit -m "update 'third_party/torch-mlir'"
+
+      - name: Create pull request
+        uses: peter-evans/create-pull-request@v5.0.1
+        with:
+          author: Roll torch-mlir <iree-github-actions-bot@google.com>
+          branch: rolltorch_mlir_${{ github.run_number }}
+          commit-message: |
+            update torch-mlir version
+          committer: Roll torch-mlir <iree-github-actions-bot@google.com>>
+          title: "[roll torch-mlir] update torch-mlir"
+          token: ${{ secrets.WRITE_ACCESS_TOKEN }}
```

Comment:
Do we want to do auto-approval?

https://github.com/openxla/openxla-pjrt-plugin/blob/97deaf61dac7826a3fefaa5d4d3598e46e558026/.github/workflows/roll_nightly_deps.yml#L78-L81

Reply:
In this situation, it will approve it but doesn't someone still have to come and merge it after seeing if checks pass?

Reply:
So I guess in that case someone is saved the few seconds of approving before merging?

Reply:
I think this line also enables the auto-merge on the PR, so once the bot approves it, the PR will be auto-merged

https://github.com/openxla/openxla-pjrt-plugin/blob/97deaf61dac7826a3fefaa5d4d3598e46e558026/.github/workflows/roll_nightly_deps.yml#L77C1-L77C1

Not sure if peter-evans/create-pull-request offers the same option

Reply:
I'm silly. Thanks. I'll just switch to using the gh cli like the one there